elasticsearch_metrics/metric.py

```diff
@@ -64,11 +74,19 @@ def get_index_template(cls):
     @classmethod
     def get_index_name(cls, date=None):
         date = date or timezone.now().date()
-        # TODO: Make dateformat configurable
-        dateformat = "%Y.%m.%d"
+        dateformat = settings.DATE_FORMAT
         date_formatted = date.strftime(dateformat)
         return "{}-{}".format(cls._template_name, date_formatted)
 
 
 class Metric(Document, BaseMetric):
     __doc__ = BaseMetric.__doc__
+
+    def save(self, using=None, index=None, validate=True, **kwargs):
```

Comment:
My previous comment got lost, so reposting:

We should probably expose a way to pass a `date`

```python
my_metric.save(date=datetime(2020, 2, 14))
```

Reply:
Why are we passing a date?  Is this the date used for the index?

Reply:
Precisely. If we are exposing a `date` parameter on `get_index_name`, then we should also expose it in `save`.

Reply:
but index is already a property on the save method.  I could pass in index `osf_metrics_preprintviews-2018-03-03` and conflicting date `2018-04-04`.

Reply:
Ah, good point. Ignore my suggestion.